API/api.py

```diff
@@ -256,12 +252,39 @@ def post_pit_scouting_pictures(data: UploadFile, team: str, event:str, year:int)
     PictureCollection.insert_one(file_data)
     return {"message": "File uploaded successfully"}
 
+@app.post("/{year}/{event}/{team}/DeletePictures/")
```

Comment:
Instead of using @app.post consider using @app.delete. Use the dedicated HTTP delete method instead of specifying deletion in the file path.